### PR TITLE
use slogger for runLauncher logging

### PR DESCRIPTION
**Pull request by James-Pickett**

### CODE REVIEW

File: cmd/launcher/launcher.go [hunk 1]
```diff
@@ -417,7 +427,10 @@ func runLauncher(ctx context.Context, cancel func(), slogger, systemSlogger *mul
 
 		if err != nil {
 			// For now, log this and move on. It might be a fatal error
-			level.Error(logger).Log("msg", "Failed to setup localserver", "error", err)
+			slogger.Log(ctx, slog.LevelError,
+				"failed to  setupup local server",
```
Comment (RebeccaMahany) [inline cmd/launcher/launcher.go:431]:
```suggestion
				"failed to setup localserver",
```

File: cmd/launcher/launcher.go [hunk 2]
```diff
@@ -558,8 +575,8 @@ func runOsqueryVersionCheck(ctx context.Context, logger log.Logger, osquerydPath
 		return
 	}
 
-	level.Debug(logger).Log("msg", "checked osqueryd version",
-		"version", outTrimmed,
+	slogger.Log(ctx, slog.LevelDebug,
+		"checked osqueryd version",
 		"execution_time_ms", executionTimeMs,
 		"osqueryd_path", osquerydPath,
```
Comment (RebeccaMahany) [inline cmd/launcher/launcher.go:581]: We don't want to include `"version", outTrimmed,`?

Comment (James-Pickett) [reply cmd/launcher/launcher.go:582]: good catch, updated

File: cmd/launcher/launcher.go (hunk 1 above)
Comment (James-Pickett) [reply cmd/launcher/launcher.go]: thank you

**Review by RebeccaMahany — APPROVED** [review]

**Pull request merged**